shell.nix

```
@@ -0,0 +1,4 @@
+{ pkgs ? import <nixpkgs> {} }:
+  pkgs.mkShell {
+    nativeBuildInputs = [ pkgs.buildPackages.nodejs-16_x ];
```

```suggestion
    buildInputs = [ pkgs.buildPackages.nodejs-16_x nodePackages.npm ];
```

Let's make sure we get npm as well. I switched `nativeBuildInputs` to `buildInputs` I'm not sure if it matters for nix shells (see https://discourse.nixos.org/t/use-buildinputs-or-nativebuildinputs-for-nix-shell/8464/4) but I think build inputs is more appropriate for a shell / it's consistent with the others

---

Sounds good! Ya I read that post too and couldn't figure out if there was a preferred one.